staging/src/k8s.io/api/core/v1/well_known_labels.go

```diff
@@ -39,4 +39,40 @@ const (
 	// controllers and kube-proxy to check if the Endpoint objects should be replicated when
 	// using Headless Services
 	IsHeadlessService = "service.kubernetes.io/headless"
+
+	// TaintNodeNotReady will be added when node is not ready
+	// and feature-gate for TaintBasedEvictions flag is enabled,
+	// and removed when node becomes ready.
+	TaintNodeNotReady = "node.kubernetes.io/not-ready"
```

Comment:
package location looks good, but I'd put these in a file named `well_known_taints.go`

Reply:
done.